packages/holocron/src/index.js

```diff
@@ -51,4 +65,16 @@ export {
   RenderModule,
   holocronModule,
   forceLoadModule,
+  validateExternal,
```

Comment:
In the `moduleRegistry` file we just export the functions inline. Can we keep that consistent?

Reply:
this is matching how we do it in the other files